WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewController.swift

```diff
@@ -126,6 +180,25 @@ private extension DashboardViewController {
 extension DashboardViewController: DashboardUIScrollDelegate {
     func dashboardUIScrollViewDidScroll(_ scrollView: UIScrollView) {
         hiddenScrollView.updateFromScrollViewDidScrollEventForLargeTitleWorkaround(scrollView)
+        showOrHideSubtitle(offset: scrollView.contentOffset.y)
+    }
+
+    private func showOrHideSubtitle(offset: CGFloat) {
+        if ServiceLocator.stores.sessionManager.defaultSite?.name != nil {
+            if offset > stackView.frame.height {
+                storeNameLabel.isHidden = true
+            }
+            if offset < -stackView.frame.height {
+                UIView.transition(with: storeNameLabel, duration: 0.2,
+                                  options: .showHideTransitionViews,
+                                  animations: { [weak self] in
+                                    guard let self = self else {
+                                        return
+                                    }
```

Comment:
Tip: According to our style guide, this can be a one-liner! :-)

Reply:
Nice to know! Thanks for the tip!